jupyterhub/spawner.py

```diff
@@ -666,21 +666,24 @@ def poll_and_notify(self):
                 self.log.exception("Unhandled error in poll callback for %s", self)
         return status
 
-    death_interval = Float(DT_MIN)
-
+    death_interval = Float(0.1)
```

Comment:
Perhaps add a brief comment on what the death_interval is. Assuming 0.1s and is used when adding random jitter.